tests/acceptance/course-page/view-course-stages-test.js

```diff
@@ -747,15 +747,13 @@ module('Acceptance | course-page | view-course-stages-test', function (hooks) {
     });
   });
 
-  test('member badge redirects to /membership', async function (assert) {
+  test('member badge redirects to /settings/billing', async function (assert) {
     testScenario(this.server);
     signInAsSubscriber(this.owner, this.server);
 
-    await catalogPage.visit();
-    await catalogPage.clickOnCourse('Build your own Redis');
-    await courseOverviewPage.clickOnStartCourse();
```

Comment:
I'd keep this as-is to stay consistent with the tests above. We could change it, but would make the change consistent across the file + would separate into another PR since it is unrelated to the change we're making here